WordPress/Classes/Utility/ContextManager.swift

```diff
@@ -24,6 +24,13 @@ public class ContextManager: NSObject, CoreDataStack, CoreDataStackSwift {
     private let storeURL: URL
     private let persistentContainer: NSPersistentContainer
 
+    /// A serial queue used to ensure there is only one writing operation at a time.
+    ///
+    /// - Note: This queue currently is not used in `performAndSave(_:)` the "save synchronously" function, since it's
+    ///   not safe to block current thread. Considering this function is going to be removed soon, I think it's okay to
+    ///   make this compromise.
```

Comment:
> Considering this function is going to be removed soon

Is the function to be removed soon is `performAndSave(_:)`

Reply:
Yes, I've rephrased this comment in https://github.com/wordpress-mobile/WordPress-iOS/pull/20060/commits/89d8a4efc58abe57c5bf7a7e956609debaa4ecdf, hopefully it's clearer now 😃

Reply:
Great, thank you!